client/me/account-close/closed.scss

```diff
@@ -0,0 +1,10 @@
+.account-close__spinner {
+	padding-top: 120px;
+	text-align: center;
+
+	&-text {
```

Comment:
Can you expand this to the full `.account-close__spinner-text`? That makes it possible to grep the codebase for CSS class names.

Reply:
Done! :)